mkdocs.yml

```diff
@@ -43,117 +43,127 @@ nav:
-          - HPCToolkit: Documentation/Development/Performance_Tools/hpctoolkit.md
-          - LIKWID: Documentation/Development/Performance_Tools/likwid.md
-          - Reveal: Documentation/Development/Performance_Tools/reveal.md
-          - Roofline Performance Model: Documentation/Development/Performance_Tools/roofline.md
-          - Timemory: Documentation/Development/Performance_Tools/timemory.md
-        - Debug Tools:
-          - ARM: Documentation/Development/Debug_Tools/arm.md
-          - GDB: Documentation/Development/Debug_Tools/gdb.md
-          - CCDB: Documentation/Development/Debug_Tools/ccdb.md
-          - Intel C Inspector: Documentation/Development/Debug_Tools/intelc.md
-          - TotalView: Documentation/Development/Debug_Tools/totalview.md
-          - Valgrind: Documentation/Development/Debug_Tools/valgrind.md
-        - Vizualization and Analytics: 
-          - FastX: Documentation/Development/Vizualization/FastX/fastx.md
-        - Jupyter:
-          - Documentation/Development/Jupyter/index.md
-          - Documentation/Development/Jupyter/jupyterhub.md
+          - Avizo:      Documentation/Applications/avizo.md
+          - ANSYS CFD : Documentation/Applications/ansys.md
+          - COMSOL:     Documentation/Applications/comsol.md
+          - FEniCS:     Documentation/Applications/fenics.md
+          - Gaussian:   Documentation/Applications/guassian.md
+          - GAMS:       Documentation/Applications/gams.md
+          - Git:        Documentation/Applications/git.md
+          - Gurobi:     Documentation/Applications/gurobi.md
+          - LAMMPS:     Documentation/Applications/lammps.md
+          - MATLAB:     Documentation/Applications/matlab.md
+          - Paraview:   Documentation/Applications/paraview.md
+          - Q-chem:     Documentation/Applications/qchem.md
+          - R:          Documentation/Applications/r.md
+          - STAR-CCM+:  Documentation/Applications/starccm+.md
+          - VASP:       Documentation/Applications/vasp.md
+          - Visit:      Documentation/Applications/visit.md
+      - Development:
+         - Languages:
+             - Julia:   Documentation/development/languages/julia/index.md
+             - C++:     Documentation/development/languages/c++/index.md
+             - R:       Documentation/development/languages/r/index..md
+             - Bash:    Documentation/development/languages/bash/index.md
+             - Fortran: Documentation/development/languages/fortran/index.md
+             - Python:
+                  - Documentation/development/languages/python/index.md
+                  - Documentation/development/languages/python/nrel_hpc_python/index.md
+                  - Documentation/development/languages/python/dask/index.md
+                  - Jupyter: Documentation/development/languages/python/jupyter/index.md
+                  - Anaconda: Documentation/development/languages/python/anaconda/index.md
```

Comment:
Anaconda is also in the Customization section. Also "Conda" is the package manager, is this intended to be "Conda" or "Anaconda"